<launch>

	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server">

		<param name="resolution" value="0.06" />
		<param name="frame_id" type="string" value="camera_link" />
		<param name="sensor_model/max_range" value="2.0" />
		<param name="latch" value="false" />
		<!-- <param name="sensor_model/miss" value="0.05" />
		<param name="sensor_model/hit" value="0.95" />
        <param name="sensor_model/min" value="0.12" />
        <param name="sensor_model/max" value="0.80" /> -->
		<remap from="cloud_in" to="/camera/depth/points_xyzrgb" />
	
	</node>

</launch>
